editoast/src/views/paced_train.rs

```diff
@@ -202,19 +211,62 @@ struct SimulationBatchForm {
 )]
 async fn simulation_summary(
     State(AppState {
-        db_pool: _db_pool,
-        valkey: _valkey_client,
-        core_client: _core,
+        db_pool,
+        valkey: valkey_client,
+        core_client: core,
```

Comment:
So much for consistency. The `AppState` is not consistent, sometimes using the suffix `_client` sometimes not... But maybe we should avoid to actually invert it completely in our functions? Let's keep the suffix everywhere for example?

```suggestion
        valkey: valkey_client,
        core_client,
```

Reply:
We should do an homogenization pass at some point